databases/week1/design.sql

```diff
@@ -0,0 +1,10 @@
+SELECT COUNT(*) AS task_count FROM tasks;
+SELECT COUNT(*) AS task_count FROM tasks WHERE due_date IS NULL ;
+SELECT * FROM tasks WHERE status = 'done';
+SELECT * FROM tasks WHERE status != 'done';
```

Comment:
Same here ☺️ 

It all depends on your database. If it has 'status' column in the tasks table with string values such as 'done', it will work fine.